src/index.js

```diff
@@ -248,6 +251,17 @@ function getMatchingElementsByClassName (elementIterator, classNames) {
   return results
 }
 
+function getMatchingElementById (elementIterator, id) {
+  let element
+  while ((element = elementIterator.next())) {
```

Comment:
I love the usage of iterators. Too bad we can't (I assume) use generators or `for of` due to IE11 support.

Reply:
We are compiling it down to ES5 using Bublé. But yeah generators tend to add a lot of overhead in that case.